hawkular-tenant-jaxrs-filter/pom.xml

```diff
@@ -64,8 +64,8 @@
 
   <dependencies>
     <dependency>
-      <groupId>javax</groupId>
-      <artifactId>javaee-api</artifactId>
+      <groupId>org.jboss.resteasy</groupId>
```

Comment:
Metrics don't depend on it and I guess it is the reason they freaked out when asked to use this instead of their own tenant handling.